examples/small_world/src/old_folder_structure/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
-load("@rules_swiftnav//cc:defs.bzl", "UNIT", "swift_cc_library", "swift_cc_test")
+load("@rules_swiftnav//cc:defs.bzl", "UNIT", "swift_cc_test")
 
-swift_cc_library(
+cc_library(
```

Comment:
Why does this not use the swift macro?

Reply:
This is to test the behavior of the toolchains, when the normal rules are used (mimic behavior from 3rd party libs)